shell/browser/api/electron_api_web_contents.cc

```diff
@@ -3708,6 +3710,30 @@ void WebContents::UpdateHtmlApiFullscreen(bool fullscreen) {
   }
 }
 
+std::string WebContents::GetMediaSourceID(
+    content::WebContents* request_web_contents) {
+  auto* frame_host = web_contents()->GetMainFrame();
+  DCHECK(frame_host);
```

Comment:
It is legal for `frame_host` to be null here (renderer crashed nor no page loaded), so this should be a if check:

```suggestion
  if (!frame_host)
    return std::string();
```

Reply:
Thanks fixed in 9cf481c